bootstrap/kubeadm/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I think you can remove this file, right?

Reply:
sure. I left it in because i left it in with docker, but I can remove it from both places. I don't think it hurts to have it though

Reply:
If we are distributing as a separate Go Module, then I think it makes sense to leave the license, since it will be downloaded along with the source when using `go get`. If it isn't a separate module, then I agree with removing.

Reply:
We won't be removing go mod in this PR, so let's leave it for now.